editor/scene_tree_dock.cpp

```diff
@@ -1141,7 +1141,7 @@ void SceneTreeDock::_notification(int p_what) {
 			node_shortcuts->add_child(button_custom);
 			button_custom->set_text(TTR("Other Node"));
 			button_custom->set_icon(get_theme_icon("Add", "EditorIcons"));
-			button_custom->connect("pressed", callable_mp(this, &SceneTreeDock::_tool_selected), make_binds(TOOL_NEW, false));
+			button_custom->connect("pressed", callable_bind(callable_mp(this, &SceneTreeDock::_tool_selected), TOOL_NEW, false));
```

Comment:
Would it be worth it (and possible) to add some sugar so we can define a `Callable` with binds directly? E.g. `callable_mp_bind(this, &SceneTreeDock::_tool_selected, TOOL_NEW, false)`?

Reply:
its a bit difficult because callable_mp is a macro, I wonder if vararg macros are finally supported in C++17..

Reply:
maybe someone with more C++17 fu can help me figure it out

Reply:
@AndreaCatania's fork has an `ecs` branch which has some examples of variadic templates, iirc. [Example](https://github.com/Andrea Catania/godot/blob/ecs/modules/ecs/iterators/query.h)

Reply:
I already implememted them in a couple of places, but it won't help here

Reply:
Also callable_bind is ok, because there are other places where regular callables (not mp) are used, so a generic function needs to exist, so given this is not used super often, it may be ok to leave as is.